zer0-os/zOS
# feat(redesign-message-bubble): update lights out bubble backgrounds

### What does this do?
- update lights out bubble backgrounds (owner and sender)

### Why are we making this change?
- redesign task

### How do I test this?
- run tests as usual
- run UI > check message bubbles against design [here](https://www.figma.com/design/pyrEsklP7OQQnC5lSV3IKq/zOS?node-id=2-1152&node-type=frame&t=sQUA0M3mRFrW2BRs-0)

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?
 

<img width="393" alt="Screenshot 2024-09-10 at 10 54 50" src="https://github.com/user-attachments/assets/86add0b3-34c1-4c7e-ae0c-eb67102964fc">
